src/qt/walletmodel.cpp

```diff
@@ -355,6 +355,9 @@ WalletModel::SendCoinsReturn WalletModel::sendCoins(WalletModelTransaction &tran
 
                 std::map<CTxDestination, CAddressBookData>::iterator mi = wallet->mapAddressBook.find(dest);
 
+                std::string key = "au"; // "au" prefix = "address used" in destdata
```

Comment:
`"used"` seems more straightforward, but more importantly, this should be moved to a `static const` in a `src/wallet/` header.